File: demo/src/main/java/com/baozi/demo/item/tbhome/HomeItemD.java
package com.baozi.demo.item.tbhome;

import android.support.annotation.NonNull;

import com.baozi.demo.R;
import com.baozi.treerecyclerview.base.ViewHolder;
import com.baozi.treerecyclerview.item.TreeItem;

/**
 * 模板D
 */
public class HomeItemD extends TreeItem {
    @Override
    public int getLayoutId() {
        return R.layout.item_tb_home_d;
    }

    @Override
    public void onBindViewHolder(@NonNull ViewHolder viewHolder) {

    }

    @Override
    public int getSpanSize(int maxSpan) {
        return maxSpan/3;
    }
}

File: demo/src/main/java/com/baozi/demo/item/tbhome/HomeItemD.kt
package com.baozi.demo.item.tbhome

import com.baozi.demo.R
import com.baozi.treerecyclerview.base.ViewHolder
import com.baozi.treerecyclerview.item.TreeItem

/**
 * 模板D
 */
class HomeItemD : TreeItem<Any>() {
    override fun getLayoutId(): Int {
        return R.layout.item_tb_home_d
    }

    override fun onBindViewHolder(viewHolder: ViewHolder) {

    }

    override fun getSpanSize(maxSpan: Int): Int {
        return maxSpan / 3
    }
}
